containers/azure-terraform/README.md

```diff
@@ -15,19 +17,24 @@
 
 ## Using this definition with an existing folder
 
-While technically optional, this definition includes the Azure Terraform extension which requires an Azure account to use. You can create a [free trial account here](https://azure.microsoft.com/en-us/free/) and find out more about using [Teraform with Azure here](https://docs.microsoft.com/en-us/azure/terraform/terraform-overview).  
+While technically optional, this definition includes the Azure Terraform extension which requires a few pre-requisites:
+
+- [Azure Subscription](https://azure.microsoft.com): A current Azure subscription or a free trail account needed.
+
+- [Terraform Extension](https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform): This optional HashiCorp Terraform VS Code extension adds syntax highlighting and other editing features for Terraform files using the Terraform Language Server.
```

Comment:
This section is confusing as it lists the Terraform Extension as a prerequisite for itself.

Reply:
Updated to remove this